reports/2022.html

```diff
@@ -1262,51 +1262,101 @@ <h3>Links</h3>
           <dt>Previous WCCG Report(s)</dt>
           <dd>N/A</dd>
           <dt>GitHub issues:</dt>
-          <dd>---</dd>
+          <dd>https://github.com/WICG/webcomponents/issues/738</dd>
           <dt>Browser positions:</dt>
-          <dd>---</dd>
+          <dd>
+            <ul>
+              <li>
+                <strong>Chrome</strong> &mdash; shipped
+              </li>
+              <li>
+                <strong>Firefox</strong> &mdash;
```

Comment:
Did we want to add a link to Firefox position here, or has it been shipped as well?

Reply:
Good call out. They haven't addressed it, I created an issue and added the link.